### PR TITLE
[improve][doc] Add instructions for enabling replicated subscription

### PR DESCRIPTION
**Pull request by Anonymitaet**
Fix https://github.com/apache/pulsar/issues/20051

- [x] `doc` 

@hangc0276 thanks for your technical explanations!

Ping @D-2-Ed for review, thanks!

### CODE REVIEW

**Comment by Anonymitaet** [comment]
Previews look good:

image

image

**Comment by Anonymitaet** [comment]
For reviewers: I'll apply the same doc changes to 2.11.x and 2.10.x if this PR LGTY.

**Comment by Anonymitaet** [comment]
@hangc0276 @xiaogang-sn Could you please review this PR from a technical perspective? TIA!

File: docs/client-libraries-cluster-level-failover.md [hunk 1]
```diff
@@ -26,7 +26,8 @@ For more information about cluster-level failover, including concepts, benefits,
   * The number of BookKeeper nodes should be equal to or greater than the ensemble quorum.
   * The number of ZooKeeper nodes should be equal to or greater than 3.
 * **Turn on geo-replication** between the primary cluster and any dependent cluster (primary to backup or backup to backup) to prevent data loss.
-* Set `replicateSubscriptionState` to `true` when creating consumers.
+* Set `enableReplicatedSubscriptions` to `true` in [`broker.conf`](https://github.com/apache/pulsar/blob/470b674016c8718f2dfd0a0f93cf02d49af0fead/conf/broker.conf#L592).
```
Comment (xiaogang-sn) [inline docs/client-libraries-cluster-level-failover.md:29]: Can we simply say "enable replicated subscription" here with the link pointing to https://pulsar.apache.org/docs/2.11.x/administration-geo/#enable-replicated-subscription which will include both broker configuration and consumer configuration?

Comment (Anonymitaet) [reply docs/client-libraries-cluster-level-failover.md]: Good catch! Updated. PTAL.

**Review by xiaogang-sn — APPROVED** [review]

**Review by hangc0276 — APPROVED** [review]

**Pull request merged**